File: api/server/src/main/java/su/plo/voice/api/server/player/VoiceServerPlayer.java
package su.plo.voice.api.server.player;

import org.jetbrains.annotations.NotNull;
import su.plo.slib.api.server.entity.player.McServerPlayer;

/**
 * Represents a voice API for the server player.
 */
public interface VoiceServerPlayer extends VoicePlayer {

    @NotNull McServerPlayer getInstance();
}

File: server/paper/src/main/kotlin/su/plo/voice/paper/extension/VoiceServerPlayer.kt
package su.plo.voice.paper.extension

import org.bukkit.entity.Player
import su.plo.voice.api.server.player.VoiceServerPlayer

/**
 * Gets a Bukkit [Player] of [VoiceServerPlayer].
 *
 * @return The Bukkit player.
 */
fun VoiceServerPlayer.asBukkitPlayer(): Player =
    instance.getInstance()
